tests/phpunit/tests/REST_API/Stories_Controller.php

```diff
@@ -140,25 +140,32 @@ public function test_filter_posts_orderby_irrelevant_query() {
 		$controller = new \Google\Web_Stories\REST_API\Stories_Controller( \Google\Web_Stories\Story_Post_Type::POST_TYPE_SLUG );
 
 		$initial_orderby = 'foo bar';
+		$initial_join    = 'foo baz';
```

Comment:
There would need to be a `test_filter_posts_join` tests as well.

However, testing that both functions return specific SQL statements doesn't guarantee that the REST API result is actually ordered. IMO the test in #2345 is preferable over this.